katello/hooks/pre/20-certs_update.rb

```diff
@@ -18,7 +18,7 @@ def mark_for_update(cert_name, hostname = nil)
 cert_file = param('certs', 'server_cert').value
 key_file  = param('certs', 'server_key').value
 
-if app_value('certs_update_server_ca') && !module_enabled?('katello')
+if app_value('certs_update_server_ca') && !katello_enabled?
   fail_and_exit("--certs-update-server-ca needs to be used with katello", 101)
 end
```

Comment:
This part should really be a `pre_commit` hook IMHO. In `pre` the answers are already saved to disk. You should fail on invalid user input in `pre`.

It should also check if the `certs` module is enabled, rather than `katello`, right? It checks if the `certs` params are passed in and I think that `server_ca_cert` can also be used on a content proxy.

Reply:
File an issue if you don't mind, touching our certs code requires a Friday :P